spec/codecs/line_spec.rb

```diff
@@ -57,11 +59,28 @@
       insist { decoded } == true
     end
 
-    it "should return an event from a valid utf-8 string" do
-      subject.decode("München\n") do |e|
-        insist { e.is_a?(LogStash::Event) }
-        insist { e.get("message") } == "München"
+    ecs_compatibility_matrix(:disabled, :v1, :v8 => :v1) do |ecs_select|
+
+      before(:each) do
+        allow_any_instance_of(described_class).to receive(:ecs_compatibility).and_return(ecs_compatibility)
+      end
+
+      let(:message) { "München" }
+      let(:line) { "#{message}\n" }
+
+      it "should return an event from a valid utf-8 string" do
+        subject.decode(line) do |e|
+          expect( e ).to be_a LogStash::Event
+          expect( e.get("message") ).to eql message
```

Comment:
Let's make the assertions match the description while we are at it.

```suggestion
          expect( e.get("message") ).to eql message
          expect( e.get("message").encoding ).to eql Encoding.find('UTF-8')
          expect( e.get("message").valid_encoding? ).to be true
```